### ISSUE TITLE
spelling error in Cartilage-hair hypoplasia annotations

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#10201]
**Bug description**
In the latest HPO annotation file (2023-10-09), the disease OMIM:250250 is spelled 'Cartilage-Hair hypoplasia' (with a capital H in 'Hair') for the annotation HP:0045075, but 'Cartilage-hair hypoplasia' (small 'h' in hair) for all the other annotations.

**Comment by pnrobinson** [comment]
As far as I can see in the original files and in the website, the spelling is with a lower-case h for hair
https://hpo.jax.org/app/browse/disease/OMIM:250250

Where did you see this?

**Comment by azankl** [comment]
I downloaded the HPO annotations (2023-10-09) from here: https://hpo.jax.org/app/data/annotations
Below is a screenshot of the file, filtered for rows with disease name 'Cartilage-Hair hypoplasia'.
The row with the different spelling is highlighted.
Might seem like a small problem, but I was developing a script that included matching on disease names, and the difference in spelling (Hair vs hair), tripped it up.

<img width="1360" alt="Screenshot 2023-12-13 at 4 22 42 pm" src="https://github.com/obophenotype/human-phenotype-ontology/assets/4812335/d0b5483f-f87f-4185-a5d8-e2b4139872bc">

**Comment by pnrobinson** [comment]
I see -- there was one type, thanks, I fixed it!
It is probably best to have scripts depend on the disease ID rather than the label though!
